alexandria/core/serializers.py

```diff
@@ -206,26 +207,34 @@ def validate(self, *args, **kwargs):
         `validator_classes` attribute of the FileViewSet.
         """
         validated_data = super().validate(*args, **kwargs)
-        if validated_data.get(
-            "variant"
-        ) != models.File.Variant.ORIGINAL and not validated_data.get("original"):
-            file_variant = validated_data.get("variant")
-            raise ValidationError(
-                f'"original" must be set for variant "{file_variant}".'
-            )
-
-        if (
-            variant := validated_data.get("variant")
-        ) == models.File.Variant.ORIGINAL and validated_data.get("original"):
+        variant = validated_data.get("variant")
+        if variant != models.File.Variant.ORIGINAL and not validated_data.get(
+            "original"
+        ):
+            raise ValidationError(_(f'"original" must be set for variant "{variant}".'))
+        elif variant == models.File.Variant.ORIGINAL and validated_data.get("original"):
             raise ValidationError(
-                f'"original" must not be set for variant "{variant}".'
+                _(f'"original" must not be set for variant "{variant}".')
             )
 
         mime_type = validated_data["content"].content_type
         if mime_type == "application/octet-stream":
-            guess, _ = guess_type(validated_data["name"])
+            guess, encoding = guess_type(validated_data["name"])
```

Comment:
i've seen people use `__` if `_` is already used, but I'm not sure if I like it or not :-D